### PR TITLE
Adjust integration name and branding

### PR DESCRIPTION
**Pull request by nikhilguptanetnologyio**
### What does this PR do?

Fixes potential branding inaccuracies and adjusts integration name accordingly.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: nn_sdwan/manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "display_name": "Netnology SD-WAN",
+  "display_name": "Netnology Cisco SD-WAN",
```
[the remark this answers is not in the document]
Comment (nikhilguptanetnologyio) [reply nn_sdwan/manifest.json]: Done!

File: .codecov.yml [hunk 2]
```diff
@@ -203,14 +211,6 @@ coverage:
         target: 75
         flags:
         - pulsar
-      PureFA:
```
[the remark this answers is not in the document]
Comment (nikhilguptanetnologyio) [reply .codecov.yml:206]: Correct me if I'm wrong, but the latest commit on master at the moment looks to be 33ebbd6d, which is what this PR works off of.

[the remark this answers is not in the document]
Comment (sarah-witt) [reply .codecov.yml:206]: Hm, could you try reverting the change in codecov.yml and see if validation still pases?

**Comment by sarah-witt** [comment]
Screen Shot 2022-06-08 at 2 19 37 PM
@nikhilguptanetnologyio Everything looks good! Once you update `.codecov.yml` this should be good to go!

**Comment by nikhilguptanetnologyio** [comment]
@sarah-witt thanks, I reverted `.codecov.yml` back to 33ebbd6 and all the checks still pass. Since we changed the readme a bit, [our integration's docs page](https://docs.datadoghq.com/integrations/nn_sdwan/) will need to be updated as well. Will the link to that change?

**Comment by sarah-witt** [comment]
@nikhilguptanetnologyio thanks! Yes, the docs page will be updated once this change is merged. The link will stay the same though: https://docs.datadoghq.com/integrations/nn_sdwan/

**Review by anawishnoff — APPROVED** [review]

**Review by sarah-witt — APPROVED** [review]

**Comment by sarah-witt** [comment]
@nikhilguptanetnologyio Let me know when you are ready for us to merge this!

**Comment by nikhilguptanetnologyio** [comment]
@sarah-witt @anawishnoff We're good to go, you can merge this. Thank you!

**Pull request merged**